test/fuzz/fuzz-unit-file/directives.service

```diff
@@ -50,6 +50,7 @@ ConditionKernelCommandLine=
 ConditionKernelVersion=
 ConditionMemory=
 ConditionNeedsUpdate=
+ConditionOSRelease=
```

Comment:
Is it not necessary to add `AssertOSRelease=`?

Reply:
Ah of course! The error mentioned the other one, so I was confused. Thanks!